src/Instrument/Transformer/CachingTransformer.php

```diff
@@ -87,10 +96,11 @@ public function transform(StreamMetaData $metadata)
         }
 
         $originalUri = $metadata->uri;
-        $cacheUri    = str_replace($this->rootPath, $this->cachePath, $originalUri);
+        $cacheUri    = $this->container->get('aspect.cache.path.resolver')->getCachePathForResource($originalUri);
```

Comment:
You should store this instance of service in the constructor. Personally, I'd like to see this implemented with DIC, but it's not possible for the moment. However, accessing local variable will be much faster than asking the container everytime.